.gitignore

```diff
@@ -2,3 +2,17 @@ java/.idea
 java/target
 java/*.iml
 .idea
+*.env
+python/__pycache__
```

Comment:
If I'm not mistaken, you can write `**/__pycache__` and it will ignore it everywhere.
Same for `build` and `dist`, and `.egg-info`

Reply:
Yup, great suggestion. Thanks Arseny!